### DISCUSSION TITLE
开启IPV6和NF共存

### DISCUSSION THREAD

**Discussion by tvrohd** [kkkgo/PaoPaoDNS#41] [问答和帮助]
用了泡泡DNS后  开V6也能看上NF了,但是不是最完美,PC端 IOS手机平板端   安卓手机端,都正常,NF和IPV6共存不影响,唯独安卓TV端,可以进NF,播放视频就提示代理,应该是没有通过DNS解析,而是直连了NF的IPV6地址,关闭IPV6 ,各个设备都正常,有什么思路办法解决安卓TV端的问题吗

**Comment by kkkgo** [comment]
你可以检查客户端DHCP分配的DNS是否只有PaoPaoDNS，如果有其他DNS，可以尝试在路由器设置里面禁止下发IPV6的DNS服务器。
